mm2src/lp_ordermatch/orderbook_depth.rs

```diff
@@ -30,6 +31,30 @@ struct PairWithDepth {
 pub async fn orderbook_depth_rpc(ctx: MmArc, req: Json) -> Result<Response<Vec<u8>>, String> {
     let ordermatch_ctx = OrdermatchContext::from_ctx(&ctx).unwrap();
     let req: OrderbookDepthReq = try_s!(json::from_value(req));
+
+    let wallet_only_pairs: Vec<_> = req
```

Comment:
This statement can be simplified to (I have included the proposed `is_wallet_only_ticker`)
```rust
    let wallet_only_pairs: Vec<_> = req
        .pairs
        .iter()
        .filter(|pair| is_wallet_only_ticker(&ctx, &pair.0) || is_wallet_only_ticker(&ctx, &pair.1))
        .collect();
```

It avoids cloning the entire `req.pairs` by getting the vector of references.